src/manifest.ts

```diff
@@ -5,43 +5,64 @@ export class Manifest {
     constructor(public name: string, public namespace: string, public workdir: string, public port: number) {}
 }
 
-export function parseManifest(parsed: yaml.Document.Parsed): Manifest[] {
-    const j = parsed.toJSON();
-    const result: Array<Manifest> = [];
+export function isDockerCompose(manifest: any): boolean {
```

Comment:
this could be a v1 okteto manifest.
@jLopezbarb what's the logic in the CLI? I think we check for the `name` property, which is mandatory in okteto manifest v1. If not, we check for `services` to check if it's a compose file

Reply:
Updated to check in this order: